testing/test_basic_widgets.py

```diff
@@ -203,13 +209,60 @@ class TestForm(View):
     with pytest.raises(TypeError):
         view.table['boom!']
 
+    row = next(view.table.rows())
+    assert row.column_1.text == 'qwer'
+
+
+def test_table_no_header(browser):
+    class TestForm(View):
+        nohead_table = Table('#without_thead')
+
+    view = TestForm(browser)
+    # attributized columns for a table withOUT thead
+    row = view.nohead_table.row(event='Some Event')
+    assert row[0].text == '27.02.2017, 12:19:30'
+    assert row['Event'].text == 'Some Event'
+    assert row.event.text == 'Some Event'
+
+    assert row.read() == {'Date': '27.02.2017, 12:19:30',
```

Comment:
This assertion is the reproduction of #103, where a table row retrieved via a kwarg filter is read